priv/repo/migrations/20160803181844_replace_username_with_slack_handle.exs

```diff
@@ -0,0 +1,35 @@
+defmodule Cog.Repo.Migrations.ReplaceUsernameWithSlackHandle do
+  use Ecto.Migration
+  alias Cog.Repo
+  alias Cog.Models.User
+
+  def change do
+    users = User
+    |> Repo.all
+    |> Repo.preload([chat_handles: :chat_provider])
```

Comment:
I'm going to make this change. But, this is really just a symptom of how we're misusing migrations to rebuild the db schema. Ideally, we'd just be storing a schema dump of the database and use migrations to alter that dump. I remember seeing a schema dump feature available in Ecto 2.0. We should research what'd take to start using it.